### ISSUE TITLE
[Enhancement]: Enable merchants to edit the "Product" and "Total" texts in the Cart block

### ISSUE THREAD

**Issue by nielslange** [woothemes/woocommerce#49559]
### Describe the solution you'd like

In https://github.com/woocommerce/woocommerce/discussions/47921#discussioncomment-10036375, @jocala99 reported that it's not possible to edit the "Product" text and shared the following screenshot:

![Screenshot 2024-07-15 at 7 32 09 AM](https://github.com/user-attachments/assets/59054c1b-94fb-4744-8a79-f24df648936d)

I looked up this issue and noticed that both the "Product" and the "Total" texts cannot be edited in the page editor, while the "Cart totals" text can be edited.

I propose enabling merchants to edit the "Product" and the "Total" texts in the editor as well.

**Comment by wavvves** [comment]
Summoning @nikkivias and @opr for feedback

**Comment by opr** [comment]
In my opinion, if the cart totals text is editable then the other two headings should be editable too.

**Comment by mikejolley** [comment]
I agree this inconsistency should be resolved by making all headings editable, with the current text being default. I will add this to the backlog.

**Comment by pierorocca** [comment]
What's the value of being able to edit "Total"? There are some things that need to be predictable for shoppers from site to site.  A large part of the order summary, consistency to a large degree is important.

**Comment by nielslange** [comment]
> What's the value of being able to edit "Total"?

A merchant might want to rename the title "Total" to "Price", as seen on Amazon's cart:

<img width="945" alt="Image" src="https://github.com/user-attachments/assets/02c8a6e4-4cf6-4d38-8c1d-589263f36b84" />

**Comment by pierorocca** [comment]
Oh wow I didn't even see the second "Total" @nielslange. I was laser focused on the order summary Total. No objections on my end.

**Comment by nielslange** [comment]
> Oh wow I didn't even see the second "Total" [@nielslange](https://github.com/nielslange). I was laser focused on the order summary Total. No objections on my end.

Thanks for your reply, @pierorocca. I'm glad to hear there are no objections from your side.

**Comment by yjailin** [comment]
@pierorocca @mikejolley  i see the issue needs design feedback.

TBH I don't think this is a high priority issue. I don't see much value to edit the "Total" label into "Price". Especially because there is the "unit price" next to the item name VS the item total in the "Total" column. It makes sense to have this "Total" label.

<img width="1536" height="663" alt="Image" src="https://github.com/user-attachments/assets/277b3bdb-7b7e-4305-8fb9-b32c83ce488d" />

**Comment by pierorocca** [comment]
Thanks @yjailin. What about the "Product" title? I can see a case for making that editable. e.g. a B2B site that only sells services or training. "Product" would be inappropriate. If you agree we can mark this one ready for engineering with the revised smaller scope.

**Comment by yjailin** [comment]
@pierorocca I think both labels should work the same way. One editable and one not, right next to each other in the same row, is going to feel broken. I'd fix both instead. "Item" covers products, services, courses, bookings, donations. And we already say "Item" in order emails and the admin order screen, so the cart is the odd one out today. If the labels work everywhere, nobody needs to edit them.
If you'd still rather give merchants control, fine, but then it should be both labels. 😊

**Comment by pierorocca** [comment]
Thanks @yoannjailin-ai. I'm ok with making them both editable as we can't account for all use cases and customizations.

Where we landed:

1. Change the default title from PRODUCT → ITEM
2. Allow both labels to be editable

Marking this ready for engineering.
